datumaro/components/extractor_tfds.py

```diff
@@ -217,12 +276,22 @@ def __call__(self, tfds_example: Any) -> str:
 )
 
 _VOC_ADAPTER = _TfdsAdapter(
-    category_transformers=[_SetLabelCategoriesFromClassLabelFeature(
-        ('objects', 'feature', 'label'))],
+    category_transformers=[
+        _SetLabelCategoriesFromClassLabelFeature(
+            ('objects', 'feature', 'label')),
+        _SaveFeatureClassList(('objects', 'feature', 'pose'), 'pose_names'),
+    ],
     data_transformers=[
         _SetImageFromImageFeature('image',
             filename_feature_name='image/filename'),
-        _AddObjectsFromFeature('objects', 'bbox', label_member='label')
+        _AddObjectsFromFeature('objects', 'bbox', label_member='label',
+            attribute_members=(
+                _AttributeMemberMapping('is_difficult', 'difficult'),
+                _AttributeMemberMapping('is_truncated', 'truncated'),
+                _AttributeMemberMapping('pose',
+                    value_converter=lambda idx, state: state.pose_names[idx]),
```

Comment:
Probably, it needs to be aligned with https://github.com/openvinotoolkit/datumaro/blob/develop/datumaro/plugins/voc_format/format.py#L50

Reply:
The only difference was the case of the string. I changed the adapter to convert the pose names to title case to better match the original dataset.